### ISSUE TITLE
Add: render email links

### ISSUE THREAD

**Issue by yosevu** [theiceshelf/firn#43]
I would like to be able to include email links in templates. What would be the best way to do this?

The first solution that comes to mind is to parse external `mailto` Org mode links e.g. `[[mailto:name@example.com][Email]]` to `<a href="mailto:name@example.com">Email</a>` and add another check [here](https://github.com/theiceshelf/firn/blob/master/clojure/src/firn/markup.clj#L367)?

It looks like another option is to write [raw HTML in Org mode](https://orgmode.org/manual/Quoting-HTML-tags.html), like this e.g.

```org
@@html:<a href="mailto:name@example.com">Email</a>@@

#+BEGIN_EXPORT html
<a href="mailto:name@example.com">Email</a>
#+END_EXPORT
```

**Comment by teesloane** [comment]
Yep - Firn would need a new regex to parse mailto links. I will add that to the issue board as a feature. 

If you are looking for alternatives in the meantime, you can write raw html directly in your org mode files (without `@@` even - for better or worse...)  and it will work (but will be without the resulting css classes that are applied as a result of parsing normal org structures.

**Comment by teesloane** [comment]
The solve for this has been pushed to master and will be available in 0.0.10.
